plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -188,9 +216,14 @@ public TableInfo getCachedTable(Duration viewExpiration, TableInfo remoteTableId
         return materializationCache.getCachedTable(this, query, viewExpiration, remoteTableId);
     }
 
+    public String getParentProjectId()
+    {
+        return this.parentProjectId;
```

Comment:
Remove redundant `this.`. Also, is it possible to use `bigQuery.getOptions().getProjectId()` instead?

Reply:
Yup, it's possible to use that instead and not assign a new `parentProjectId` variable.